sci-mathematics/yafu/files/yafu-1.34.3-seds.patch

```diff
@@ -0,0 +1,57 @@
+Weres seds from ebuild:
+	sed -i \
+		-e 's:../gmp/include:gmp:' \
+		-e 's:../gmp-ecm/include:gmp-ecm:' \
+		-e 's:# LIBS += -L../msieve/lib/linux/x86_64:LIBS += -lmsieve -lz -ldl:' \
+		-e 's:CFLAGS = -g:#CFLAGS = -g:' \
+		-e '/$(LIBS)$/s:$(CC):$(CC) $(LDFLAGS):g' Makefile || die
+	sed -i -e 's:\"config.h\":<gmp-ecm/config.h>:g' top/driver.c || die
+
+	# proper ggnfs default path
+	sed -i -e 's~strcpy(fobj->nfs_obj.ggnfs_dir,"./");~strcpy(fobj->nfs_obj.ggnfs_dir,"/usr/bin/");~' factor/factor_common.c || die
+--- a/Makefile
++++ b/Makefile
+@@ -20,16 +20,16 @@
+ #CC ?= gcc
+ #CC = x86_64-w64-mingw32-gcc-4.5.1
+ #CFLAGS = -march=core2 -mtune=core2
+-CFLAGS = -g
```

Comment:
It's an antipattern usually to have a patch which _adds_ a comment line, because the patch already preserves removed lines as `-` in the diff. But I understand that the comment here is kind of to make it more idiomatic / fitting with the upstream Makefile, even if we don't submit it.

i.e. I don't insist you change it here, but keep it in mind in general.